### PR TITLE
Add new job submission limits to policy page

### PR DESCRIPTION
**Pull request by Comeani**

### CODE REVIEW

File: docs/policies/job-scheduling-policy.md [hunk 1]
```diff
@@ -166,6 +166,22 @@ that has been marked as a dependency.</td>
...
 ### Reasons related to exceeding a usage limit:
+
+#### JobArrayTaskLimit, QOSMaxJobsPerUserLimit and QOSMaxJobsPerAccountLimit
+One or more of your jobs have exceeded limits in place on the number of jobs you can have in the queue 
+**that are actively accruing priority**. Jobs with this status will remain in the queue, but will not being accruing
+priority until other jobs from the submitting user have completed. 
+
+In most cases the **per-account limit is 500 jobs**, and the **per-user limit is 100 jobs**. You can use 
+`sacctmgr show qos format=Name%20,MaxJobsPA,MaxJobsPU,MaxSubmitJobsPA,MaxSubmitJobsPU,MaxTresPA%20` to view the limits 
+for any given QOS. The maximum job array size is 100. These are slightly higher on HTC.
```
Comment (chnixi) [inline docs/policies/job-scheduling-policy.md:177]: Can you provide the exact array size limit on HTC as well?

**Review by chnixi — COMMENTED** [review]
Since we got rid of smp-legacy QOS, should the table on SMP QOS be updated as well?

**Comment by Comeani** [comment]
@chnixi The smp-legacy-(s,n,l) QOS was technically reinstated with the other deleted QOS. 

```

[nlc60@login1 ~] : sacctmgr show qos format=name%40 | grep smp
                               smp-smp-s 
                          smp-high-mem-s 
                          smp-high-mem-l 
                          smp-high-mem-n 
                               smp-smp-l 
                               smp-smp-n 
                            smp-legacy-s 
                            smp-legacy-n 
                            smp-legacy-l 
                        smp-smp-s-invest 
                   smp-high-mem-s-invest 
                   smp-high-mem-l-invest 
                   smp-high-mem-n-invest 
                        smp-smp-l-invest 
                        smp-smp-n-invest 
                     smp-legacy-s-invest 
                     smp-legacy-n-invest 
                     smp-legacy-l-invest 
                               smp-amd-s 
                               smp-amd-n 
                               smp-amd-l 
                        smp-amd-s-invest 
                        smp-amd-n-invest 
                        smp-amd-l-invest 
                         smp-scavenger-l 
                         smp-scavenger-n 
                         smp-scavenger-s 
                              smp-smp-ll 
                       smp-smp-ll-invest 
                         smp-high-mem-ll 
                  smp-high-mem-ll-invest
```

I don't think there is much harm in leaving it in there for now until we actually move forward with removing them. At that point, we can update the table to accurately reflect the removals.

**Review by chnixi — APPROVED** [review]
Looks good.

**Pull request merged**